docs/sql-ref-syntax-aux-cache-clear-cache.md

```diff
@@ -19,4 +19,16 @@ license: |
   limitations under the License.
 ---
 
-**This page is under construction**
+### Description
+Clear the RDD cache associated with a SQLContext.
```

Comment:
Is it worth noting that this clears _all_ cached RDDs (right)? and it's RDDs and DataFrames? I'd back-tick `SQLContext` for consistency, but might we say `SparkSession` instead?